src/functionalTest/resources/conf/routes/checkout-operation.conf

```diff
@@ -0,0 +1,23 @@
+routingOperations = ${routingOperations} [
```

Comment:
why do we need this? can we use the same task name in all cases? then we can have one configuration here for all cases.